sonic-utilities-tests/aclshow_test.py

```diff
@@ -0,0 +1,194 @@
+import sys
+import os
+from StringIO import StringIO
+import mock
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+sys.path.insert(0, modules_path)
+
+from imp import load_source
+load_source('aclshow', scripts_path+'/aclshow')
+from aclshow import *
+
+import mock_tables.dbconnector
+
+default_output = ''+ \
```

Comment:
please add a description of the command used and the parameters to invoke this output. it will help so see the big picture

Reply:
Added description to the expected outputs, ant to each test case.

Reply:
Added description to each test case, and the same description